<?xml version="1.0"?>
<launch>

    <arg name="map_file"/>

    <!-- map server -->
    <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)">
        <param name="frame_id" value="map"/>
    </node>

</launch>